### PR TITLE
Close #17 label var names

### LINKED ISSUE #6: Update and reconfigure variable labels doc
Spit into 2 pages - reasoning/explanation and reference material.

### LINKED ISSUE #17: Discuss and add: PDF label documentation reference material
Would appreciate getting input before diving in so there's less change of having to completely change it all later. This is the basic structure I'm thinking of:

## People
| PDF label | Interview order | Attachment | Document output format |
|:-|:-|:-|:-|
| user | user[0] | user[0] | **First M. Last** - name of the first user |
| other_party | other_parties[0] | other_parties[0] | **First M. Last** - name of the first other_party |

(for all reserved people)

## Other objects
| PDF label | Interview order | Attachment | Document output format |
|:-|:-|:-|:-|
| court | courts[0] | courts[0] | **Court Name** - name of first court |
| docket_number | docket_numbers[0] | docket_numbers[0] | **123456** - first docket number |

## Repeat a value in multiple places
| PDF label | Interview order | Attachment | Document output format |
|:-|:-|:-|:-|
| child__2 |  |  |  |
| child__3 |  |  |  |

## Multiple items of one category
| PDF label | Interview order | Attachment | Document output format |
|:-|:-|:-|:-|
| court2 |  |  |  |
| docket_number3 |  |  |  |

## Print all names in a group
Like people, courts, or docket numbers

| PDF label | Interview order | Attachment | Document output format |
|:-|:-|:-|:-|
| courts |  |  |  |
| docket_numbers |  |  |  |
| spouses |  |  |  |
| plaintiff3<strong>_phones</strong> | plaintiffs[2]<strong>.phone_numbers()</strong> |  |  |

## Suffixes
| PDF label | Interview order | Attachment | Document output format |
|:-|:-|:-|:-|
| parent2<strong>_name</strong> | parents<strong>.gather()</strong> | parents[1]<strong>.name</strong> | **First M. Last** - name of second parent |
| parent2<strong>_gender</strong> | parents[1]<strong>.gender</strong> | parents[1]<strong>.gender</strong> | **female** - gender of second parent |
| court<strong>_division</strong> |  |  |  |

## DOCX only
Haven't tried this yet

### PR DESCRIPTION
**Pull request by plocket**
Closes #17 and closes #6, I think. At least for the first go-through. Accidentally adds draft label variables overview doc. I can remove if needed.

Test:
- [x] The thing runs.

### CODE REVIEW

File: docs/doc_vars_reference.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+slug: /label_variables
+---
+
+Lables and variables you can use in your PDFs and DOCXs to make the most of the [AssemblyLine Weaver interview code generator](https://github.com/suffolkLITLab/docassemble-assemblylinewizard). It's more work now, but it should save you a bunch of work in the future.
+
+:::info
+This is a subset of the variables in the AssemblyLine library. These are just the words that the [ALWeaver](https://github.com/suffolkLITLab/docassemble-assemblylinewizard) knows. They help you generate your starting code.
+:::
+
+## Prefixes: [Reserved words](#reserved_words) for objects
```
Comment (BryceStevenWilley) [inline docs/doc_vars_reference.md]: I don't think that `reserved_words` (here and a few other lines later on) actually links to anything yet, is that just missing?

File: docs/doc_vars_reference.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+:::caution
+The list of `courts` will be soon be replaced with one single `trial_court` object
+:::
+
+| Document output format | PDF label | Attachment | Interview order |
+|:-|:-|:-|:-|
+| **Court Name** - name of the 1st court | court | courts[0] | courts[0] |
+| **123456** - 1st docket number | docket_number | docket_numbers[0] | docket_numbers[0] |
+|  | signature_date |  |  |
```
Comment (BryceStevenWilley) [inline docs/doc_vars_reference.md:54]: I'd either remove / comment out, or fill this in.

File: sidebars.js [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
+      type: 'category',
+      label: 'PDFs and DOCXs',
+      items: [
+        'pdfs',
+        'document_variables_reference',
+      ],
+    },
+    'github',
+    'docs_style_guide',
+    'pdfs',
```
Comment (BryceStevenWilley) [inline sidebars.js:17]: Do we want the PDF page to be linked in two different places?

File: docs/doc_vars_reference.md (hunk 1 above)
Comment (BryceStevenWilley) [reply docs/doc_vars_reference.md]: It should be `reserved-words`, I fixed in ece85eb

**Review by BryceStevenWilley — APPROVED** [review]
LGTM, only issue is that the label page is linked twice in the sidebar.

**Comment by plocket** [comment]
Good eye! We should make those issues and then fix them as we go. We'll probably need to do a fair amount of proof reading when we start properly reviewing documents.

**Pull request merged**